rfcs/rfc-docs-review-permissions.md

```diff
@@ -0,0 +1,121 @@
+# RFC: Require a docs reviewer and a technical-reviewer for PRs to `o3de.org`
+
+## Summary
+
+This RFC proposes that submissions to the `o3de/o3de.org` repo, in the `/content/docs` and `/content/static/images` directories, 
+require at least one docs reviewer and at least one technical reviewer to review and approve the PR before it can be merged. 
+
+Informally, some of the `o3de.org` community already practice this, however, the `o3de.org` repo's configuration settings don't 
+clearly reflect this behavior. The current PR process for `o3de.org` repo looks like this: 
+  
+  Two valid approvals are required before a PR that's against `main` or `development` branches can be merged. They can be from 
+  any of the teams that have [write role](https://docs.github.com/en/organizations/
+  managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization) or above. This allows PRs that 
+  have received two approvals from only docs reviewers or only technical reviewers to be merged.
+
+This RFC proposes a solution to better enforce the requirement for one docs reviewer and one technical reviewer.
+
+
+### What is the motivation for this suggestion?
+
+*Why is this important?*
+
+Requiring a docs reviewer and a technical reviewer can ensure that incoming PRs for O3DE Documentation is reviewed by the 
+appropriate people. Furthermore, limiting the reviewers to one docs reviewer and one technical reviewer at minimum can ensure 
+efficient use of resources (reviewers). 
+
+*What are the use cases for this suggestion?*
+
+Any PR in the `o3de.org` repo that amends the `/content/docs/`  or `/content/static/images` directory by updating existing docs 
+or creating new docs and supplemental media, such as images and videos. For example, a PR that adds documentation about a feature. 
+
+*What should the outcome be if this suggestion is implemented?*
+
+
+For a given PR, as described in the previous use case, 2 approvals at minimum are required before the PR can be merged: one from 
+a docs reviewer and one from technical reviewer. A docs reviewer has the expertise to ensure the PR is written well, and a 
+technical reviewer has the expertise to ensure the PR is technically accurate. 
+
+
+### Suggestion design description
+
+#### A CODEOWNERS file
+
+The `o3de.org:main` and `o3de.org:development` branches shall contain a CODEOWNERS file that dictates the `docs-reviewers` and 
```

Comment:
IMHO this needs wider circulation, have not heard about the .codeowners file for docs nor about mechanisms to own/update it.